### DISCUSSION TITLE
Is it possible to manually provide projects to discover release notes or URLs?

### DISCUSSION THREAD

**Discussion by patrickhuy** [renovatebot/renovate#9449] [Request Help] [answered]
Is it possible to override in package rules for specific packages where release notes or URLs are discovered?
For helm chart dependency updates oftentimes the "application" github repository is discovered/linked instead of the helm chart repository.

Likewise for projects with non public source code locations no link is discovered.

Is there a way for me to provide links and a strategy to retrieve release notes manually?

**Comment by viceice** [comment, marked as answer]
Yes, i use package rules, checkout [this](https://github.com/visualon/renovate-config/blob/main/meta.json#L7)  config for samples

**Reply by patrickhuy** [reply]
Thanks, very cool!
Where can we learn that this is supported? https://docs.renovatebot.com/configuration-options/ does not list it.
